chore: release v0.24.5 (#1465)
paritytech/jsonrpsee: CHANGELOG.md

[Keep a Changelog]: http://keepachangelog.com/en/1.0.0/

## [v0.24.5] - 2024-09-26

> This is patch release that mainly fixes a compilation issue when using the `tower/util` feature was not enabled.

Author: ```suggestion
This is patch release that mainly fixes a compilation issue for the server because the feature `tower/util` was not enabled.
```

> This is patch release that mainly fixes a compilation issue for the server because the feature `tower/util` was not enabled.

Author: ```suggestion
This is a patch release that mainly fixes a compilation issue for the server because the feature `tower/util` was not enabled.
```
